### PR TITLE
(Instance/Culling of Stratholme/Trial of Crusader) removed duplicated gossip_menus, fixed hardcoded texts

### PR DESCRIPTION
**Pull request by chaosua**
Culling of Stratholme Closes https://github.com/azerothcore/azerothcore-wotlk/issues/18515 
 removed duplicated gossip_menus, fixed hardcoded texts

Trial of Crusader
fixed hardcoded text from Jaraxxus text line
boss_twin_valkyr: removed unnecessary code for random txt line select (already exist groupid 6 id 0 and 1 in creature_text)
added ruRU locale for Acidmaw and Dreadscale GroupID 2

## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [x] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

## Issues Addressed:

- Closes 

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [ ] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [ ] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1.
2.
3.

## Known Issues and TODO List:

- [improve naming of variables]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by chaosua** [comment]
accidently commited trial of Crusader same type fixes in wrong branch so this PR become 2 in 1 O.o

**Comment by heyitsbench** [comment]
SQL updates are in the wrong directory, they should be in `data/sql/updates/pending_db_world`.

**Comment by pangolp** [comment]
```
ERROR 1064 (42000) at line 2 in file: '/home/runner/work/azerothcore-wotlk/azerothcore-wotlk/data/sql/updates/db_world/2024_03_10_01.sql': You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '﻿DELETE FROM `gossip_menu_option` WHERE (`MenuID`='9610') AND (`OptionID`='1')' at line 1

Applying of file '/home/runner/work/azerothcore-wotlk/azerothcore-wotlk/data/sql/updates/db_world/2024_03_10_01.sql' to database 'acore_world' failed! If you are a user, please pull the latest revision from the repository. Also make sure you have not applied any of the databases with your sql client. You cannot use auto-update system and import sql files from AzerothCore repository with your sql client. If you are a developer, please fix your sql query.
Could not update the World database, see log for details.
```

File: data/sql/updates/pending_db_world/culling_of_stratholme.sql [hunk 1]
```diff
@@ -1,7 +1,7 @@
-﻿DELETE FROM `gossip_menu_option` WHERE (`MenuID`='9610') AND (`OptionID`='1');
-DELETE FROM `gossip_menu_option` WHERE (`MenuID`='9611') AND (`OptionID`='1');
-DELETE FROM `gossip_menu_option` WHERE (`MenuID`='9612') AND (`OptionID`='1');
-DELETE FROM `gossip_menu_option` WHERE (`MenuID`='11277') AND (`OptionID`='1');
+﻿DELETE FROM `gossip_menu_option` WHERE `MenuID`=9610 AND `OptionID`=1;
```
Comment (heyitsbench) [inline data/sql/updates/pending_db_world/culling_of_stratholme.sql:1]:
```suggestion
﻿DELETE FROM `gossip_menu_option` WHERE `MenuID` IN (9610, 9611, 9612, 11277) AND `OptionID`=1;
```
https://www.azerothcore.org/wiki/sql-standards#compact-queries

**Comment by pangolp** [comment]
Apparently I was wrong. I'll see if I can investigate more.

**Comment by pangolp** [comment]
When using single quotes, and in the presence of the \', it is strange, but the sentences look strange. On the other hand, if you use double quotes, it is already displayed correctly. I don't know if that is the reason, but the truth is, I don't see anything strange in the sentence.

image

image

That change catches my attention, but perhaps it is something aesthetic, and has nothing to do with the failure.

**Comment by Elmsroth** [comment]
> When using single quotes, and in the presence of the ', it is strange, but the sentences look strange. On the other hand, if you use double quotes, it is already displayed correctly. I don't know if that is the reason, but the truth is, I don't see anything strange in the sentence.
> 
> image
> 
> image
> 
> That change catches my attention, but perhaps it is something aesthetic, and has nothing to do with the failure.

If you use VScode it is due to Mysql formatting. You can add a specific Extension for MySQL formating strings with simple quotes that will display correctly with the escaping "\"

**Comment by pangolp** [comment]
I was previewing it in the same editor here on github, which I think uses something similar to visual studio code, so I downloaded it. Also, I'm looking at other queries, and it is well written. I don't know why it fails.

**Comment by chaosua** [comment]
finaly "All checks have passed"

**Review by elthehablo — APPROVED** [review]

**Pull request merged**